core/common/src/main/java/alluxio/exception/ExceptionMessage.java

```diff
@@ -208,7 +208,8 @@
   UNKNOWN_LINEAGE_FILE_STATE("Unknown LineageFileState: {0}"),
 
   // client
-  DIFFERENT_MASTER_ADDRESS("Master address {0} is different from that in file system context {1}"),
+  DIFFERENT_CONNECTION_DETAILS(
```

Comment:
as I said, I didn't see it used.

Reply:
It seems that I missed pushing one commit, I will push that commit soon, thanks!